desktop-tool/src/web_server.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import threading
+from http import server
+from pathlib import Path
+
+from src import constants
+
+
+class _Handler(server.BaseHTTPRequestHandler):
+    def do_GET(self):
+        self.send_response(200)
+        self.send_header("Content-Type", "text/html")
+        self.end_headers()
+        self.wfile.write(
+            Path(__file__).joinpath("../..").joinpath(constants.POST_LAUNCH_HTML_FILENAME).resolve().read_bytes()
+        )
+
+    def log_request(self, code: int | str = "-", size: int | str = "-") -> None:
```

Comment:
to maintain compatibility with windows 7 for as long as possible, we're still on python 3.9 - so you'll need to use old-style typing here:

```suggestion
    def log_request(self, code: Union[int, str] = "-", size: Union[int, str] = "-") -> None:
```